proto/agent/v1/cmds.proto

```diff
@@ -19,4 +19,70 @@ message CancelPendingRequest {
 
     // optional might be usefull for debugging purposes
     string trace_id = 3;
-}
\ No newline at end of file
+}
+
+// AgentCommand is sent from the Cloud to the Agent at `/agent/{claim_id}/inbound/v1/cmd/AgentCommand`
+// the message includes the resource that the Cloud needs to GET from the Agent HTTP API along with related metadata
+message AgentCommand {
+    // currently statically set to `http`
+    string type = 1;
+    // currently statically set to `2`
+    uint32 version = 2;
```

Comment:
@underhood does the Agent still need those two fields or we can omit them?

Reply:
```
    // Originally we were expecting to have multiple types of 'cmd' message,
    // but after the new protocol was designed we will ever only have 'http'
```
so type is not needed

Reply:
`version` is now hardcoded to 2 (before we had 2 versions but version 1 has been removed from agent already)
therefore it can be omitted

Reply:
Thanks, will remove them 👍🏼